cli/version/version_tools.go

```diff
@@ -0,0 +1,140 @@
+package version
+
+import (
+	"fmt"
+	"regexp"
+	"sort"
+
+	"github.com/tarantool/tt/cli/util"
+)
+
+type ReleaseType uint16
+
+const (
+	TypeNightly ReleaseType = iota
+	TypeAlpha
+	TypeBeta
+	TypeRC
+	TypeRelease
+)
+
+type Release struct {
+	Type ReleaseType
+	Num  uint64
+}
+
+type Version struct {
```

Comment:
add descriptions to structs, also in my opinion maybe rename to TarVersion or TarantoolVersion or rename module to tar_version_tools to prevent any confusions in future.

Reply:
discussed about name, have no questions